src/Internals/measures/structure.jl

```diff
@@ -261,12 +249,15 @@ true
 See also [`top_predators`](@ref) and [`trophic_classes`](@ref).
 """
 function trophic_levels(A::AbstractMatrix)
-    A = Bool.(A)
-    trophiclvl_dict = trophic_level(UnipartiteNetwork(A))
-    trophiclvl_val = trophiclvl_dict |> values |> collect
-    trophiclvl_keys = trophiclvl_dict |> keys
-    species_idx = parse.(Int64, [split(t, "s")[2] for t in trophiclvl_keys])
-    trophiclvl_val[sortperm(species_idx)]
+    A = Matrix(A) # Ensure A is dense for inversion.
```

Comment:
No it cannot, I had an error because of that. (Because an inverse of sparse matrix is usually not sparse so they warn against inverting a sparse matrix).